src/StimulusBundle/doc/index.rst

```diff
@@ -203,6 +204,30 @@ If you want to set CSS classes:
         Hello
     </div>
 
+And with outlets:
+
+.. code-block:: html+twig
+
+    <div {{ stimulus_controller('chart', { 'name': 'Likes', 'data': [1, 2, 3, 4] }, { 'loading': 'spinner' }, { 'other': '.target' ) }}>
+        Hello
+    </div>
+
+    <!-- would render -->
```

Comment:
-would+will ?

Reply:
I just followed the other examples (that I didn't write); should I change them all?

Reply:
I dont know :/
I was just reviewing unitary lol